### DISCUSSION TITLE
How can i move from Primefaces 6.2 to 7.0.14 for a vulnerability fix

### DISCUSSION THREAD

**Discussion by richakhurana19** [primefaces/community#2545] [PrimeFaces] [answered]
How can i move from Primefaces 6.2 to 7.0.14 for a vulnerability fix? I am ready to purchase the jar. When i wrote to the mailbox contact@primetek.com.tr, the mail bounced back. I need to understand the pricing at the earliest.

**Comment by jepsar** [comment, marked as answer]
If you are concerned with security, please upgrade your PrimeFaces version to the latest version and not just to 7 (which is really old). You can update to the latest LTS (13 at this point) or to the community version (14 at this point).

Migrating is explained in https://primefaces.github.io/primefaces/14_0_0/#/../migrationguide/migrationguide

For more info on LTS, please see https://www.primefaces.org/showcase/lts.xhtml
